client/src/components/Header.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from "react";
+import styled from "@emotion/styled";
+import menuSrc from "../assets/menuIcon.svg";
+import goBackSrc from "../assets/arrowBack.svg";
+import goBackInvisibleSrc from "../assets/arrowBackInvisible.svg";
+import PropTypes from "prop-types";
+
+function Header({ title, open, setOpen }) {
+  const [displayIcon, setDisplayIcon] = useState("");
+
+  useEffect(() => {
+    if (title === "Grocery Lists") {
+      setDisplayIcon(false);
+    } else {
+      setDisplayIcon(true);
+    }
+  }, [title]);
+  return (
+    <HeaderWrapper>
+      {displayIcon && (
+        <img
+          src={goBackSrc}
+          alt="Go back Icon"
+          onClick={() => window.history.back()}
+        />
+      )}
+      {!displayIcon && <img src={goBackInvisibleSrc} alt="Go back Icon" />}
+      <h1>{title}</h1>
+
+      <img
+        src={menuSrc}
+        alt="Menu Icon"
+        open={open}
+        onClick={() => setOpen(!open)}
+      />
+    </HeaderWrapper>
+  );
+}
+
+export default Header;
+Header.propTypes = {
+  title: PropTypes.string,
+  open: PropTypes.bool,
+  setOpen: PropTypes.func,
+};
+const HeaderWrapper = styled.header`
+  background-color: #f27649;
```

Comment:
It might be an option to put your colors into a different theme file, so if you decide to change your color you don't have to do it in every file.

Reply:
CSS-variables for the orange, etc would be helpful 😄

Reply:
Good point. I will work on global style in my next step and this is when I want to create my theme. 👍